assets/js/scripts.js

```diff
@@ -198,72 +204,83 @@ function clockTick() {
     hourCycle: Settings.isClock24Hour ? 'h23' : 'h12',
   };
 
-  $('#time-in-game').text(gameTime.toLocaleString(Settings.language, clockFormat));
+  const timeInGame = document.getElementById('time-in-game');
+  if (timeInGame)
+    timeInGame.textContent = gameTime.toLocaleString(Settings.language, clockFormat);
 
-  $('.day-cycle').css('background', `url(assets/images/${nightTime ? 'moon' : 'sun'}.png)`);
+  document.querySelector('.day-cycle').style.background = `url(assets/images/${nightTime ? 'moon' : 'sun'}.png)`;

```

Comment:
This line throws an error in preview mode, because clock is not loaded in HTML dom. 
You can enable preview mode adding query string "?q=trains".